tests/test_kr.py

```diff
@@ -0,0 +1,205 @@
+    (1000000, 'uwa 1000000')
+)
+
+TEST_CASES_TO_CURRENCY_EUR = (
+    (1.00, 'iyero rimwe n\' ibice zeru'),
+    (2.01, 'amayero abiri n\'igice kimwe'),
+    (8.10, 'amayero umunani n\' ibice icumi'),
+    (12.26, 'amayero cumi n\' abiri n\' ibice makumyabiri na bitandatu'),
+    (21.29, 'amayero makumyabiri n\' ibice makumyabiri n\' icyenda'),
+    (81.25, 'amayero mirongo inani n\' ibice makumyabiri na bitanu'),
+    (100.00, 'amayero ijana n\' ibice zeru'),
+)
+
+TEST_CASES_TO_CURRENCY_FRW = (
+    (1.00, 'ifaranga rimwe n\' ibice zeru'),
+    (2.01, 'amafaranga abiri n\'igice kimwe'),
+    (8.10, 'amafaranga umunani n\' ibice icumi'),
+    (12.26, 'amafaranga cumi n\' abiri n\' ibice makumyabiri na bitandatu'),
+    (21.29, 'amafaranga makumyabiri n\' ibice makumyabiri n\' icyenda'),
+    (81.25, 'amafaranga mirongo inani n\' ibice makumyabiri na bitanu'),
+    (100.00, 'amafaranga ijana n\' ibice zeru'),
+)
+
+TEST_CASES_TO_CURRENCY_USD = (
+    (1.00, 'idolari rimwe n\' amacenti zeru'),
+    (2.01, 'amadolari abiri n\'amacenti kimwe'),
+    (8.10, 'amadolari umunani n\' amacenti icumi'),
+    (12.26, 'amadolari cumi n\' abiri n\' amacenti makumyabiri na bitandatu'),
+    (21.29, 'amadolari makumyabiri n\' amacenti makumyabiri n\' icyenda'),
+    (81.25, 'amadolari mirongo inani n\' amacenti makumyabiri na bitanu'),
+    (100.00, 'amadolari ijana n\' amacenti zeru'),
+)
+
+
+class Num2WordsENTest(TestCase):
+    def test_ordinal_special_joins(self):
+        # ref https://github.com/savoirfairelinux/num2words/issues/18
+        self.assertEqual(
+            num2words(5, ordinal=True, lang='kr'), "uwa gatanu"
+        )
+        self.assertEqual(
+            num2words(35, ordinal=True, lang='kr'), "uwa mirongo itatu na gatanu"
+        )
+        self.assertEqual(
+            num2words(9, ordinal=True, lang='kr'), "uwa cyenda"
+        )
+        self.assertEqual(
+            num2words(49, ordinal=True, lang='kr'), "uwa mirongo ine n\' icyenda"
+        )
+
+    def test_number(self):
+        for test in TEST_CASES_CARDINAL:
+            self.assertEqual(num2words(test[0], lang='kr'), test[1])
+
+    def test_ordinal(self):
+        for test in TEST_CASES_ORDINAL:
+            self.assertEqual(
+                num2words(test[0], lang='kr', ordinal=True),
+                test[1]
+            )
+
+    def test_ordinal_num(self):
+        for test in TEST_CASES_ORDINAL_NUM:
+            self.assertEqual(
+                num2words(test[0], lang='kr', to='ordinal_num'),
+                test[1]
+            )
+
+    def test_currency_eur(self):
+        for test in TEST_CASES_TO_CURRENCY_EUR:
+            self.assertEqual(
+                num2words(test[0], lang='kr', to='currency', currency='EUR'),
+                test[1]
+            )
+
+    def test_currency_frf(self):
+        for test in TEST_CASES_TO_CURRENCY_FRW:
+            self.assertEqual(
+                num2words(test[0], lang='kr', to='currency', currency='FRW'),
```

Comment:
currency should be "RWF"

Reply:
fixed! thanks for the catch, i had used the francophone notation.